src/messaging/ExchangeContext.h

```diff
@@ -214,6 +214,47 @@ class DLL_EXPORT ExchangeContext : public ReliableMessageContext,
      */
     bool IsSendExpected() const { return mFlags.Has(Flags::kFlagWillSendMessage); }
 
+    /**
+     * Notifies the exchange that our consummer is going to send a message where it knows the peer (receiver) should be active
+     *
+     * This API is used to set the IsPeerLikelyActiveHint flag.
+     * The IsPeerLikelyActiveHint flag allows the consummer of the exchange to notify the exchange that the peer (receiver)
+     * will be active for the next message the consummer will send.
+     * This allows the consummer to notify the exchange on the state of the peer with the context of the messages being sent.
+     *
+     * The IsPeerLikelyActiveHint flag is only used when calculating the next retransmission time in the ReliableMessageMgr.
+     * If the IsPeerLikelyActiveHint flag is true, we will used the Active Retransmission timeout when calculating the next
```

Comment:
`  If the IsPeerLikelyActiveHint flag is true, we will **use** the Active Retransmission timeout when calculating the next`